workflow/rules/common.smk

```diff
@@ -459,6 +461,76 @@ def get_adapters(wildcards):
     return config["adapters"]["illumina-revelo"]
 
 
+def get_final_assemblies(wildcards):
+    if wildcards.assembly_type == "masked-assembly":
+        pattern = "results/{{date}}/contigs-masked/{sample}.fasta"
+    elif wildcards.assembly_type == "pseudo-assembly":
+        pattern = "results/{{date}}/contigs-masked/{sample}.fasta"
```

Comment:
Is it on purpose to use the same path here?

Reply:
Nah, that was a copy-paste mistake. Fixed in 874adcf.